fjord/feedback/views.py

```diff
@@ -72,11 +72,18 @@ def thanks(request):
             pass
 
     if feedback:
-        product = feedback.product
+        product_db_name = feedback.product
         suggestions = get_suggestions(feedback, request)
 
-    return render(request, 'feedback/thanks.html', {
-        'product': product,
+    product = models.Product.objects.get(db_name=product_db_name)
+
+    template = 'feedback/thanks.html'
+
+    if product.slug in PRODUCT_OVERRIDE:
+      template = PRODUCT_OVERRIDE[product.slug]['thanks_template']
```

Comment:
None of those screenshots show the thank you page. I thought you were just going to use the Input thank you page as is. It'd be a lot easier to do that.

I'm not sure I'd (ab)use the `PRODUCT_OVERRIDE` thing for this, though. It'd be better to fix the way the pieces connect together than do workarounds like this.

Regarding style issues, we have an editorconfig thing. If you can configure your editor to use that, it sometimes makes it easier. http://editorconfig.org/